azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -289,23 +294,14 @@ def send_message(self, message):
         This is a synchronous event, meaning that this function will not return until the event
         has been sent to the service and the service has acknowledged receipt of the event.
 
-        If the connection to the service has not previously been opened by a call to connect, this
-        function will open the connection before sending the event.
-
         :param message: The actual message to send. Anything passed that is not an instance of the
             Message class will be converted to Message object.
         :type message: :class:`azure.iot.device.Message` or str
 
-        :raises: :class:`azure.iot.device.exceptions.CredentialError` if credentials are invalid
-            and a connection cannot be established.
-        :raises: :class:`azure.iot.device.exceptions.ConnectionFailedError` if a establishing a
-            connection results in failure.
-        :raises: :class:`azure.iot.device.exceptions.ConnectionDroppedError` if connection is lost
-            during execution.
-        :raises: :class:`azure.iot.device.exceptions.OperationTimeout` if connection attempt
-            times out
```

Comment:
What do I get if I can make send_message raise these exceptions?

Reply:
I bet you $100 you can't make it raise OperationTimeout.

Not playing the ConnectionDroppedError odds though